### PR TITLE
[Docker] Add Train requirements to ray-ml docker image

### PR DESCRIPTION
**Pull request by amogkam**
## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: docker/ray-ml/Dockerfile [hunk 1]
```diff
@@ -8,6 +8,7 @@ COPY requirements_dl.txt ./
 COPY requirements_ml_docker.txt ./
 COPY requirements_rllib.txt ./
 COPY requirements_tune.txt ./requirements_tune.txt
+COPY requirements_train.txt ./
```
Comment (matthewdeng) [inline docker/ray-ml/Dockerfile:11]: Keep this in alphabetical order?

**Review by matthewdeng — APPROVED** [review]

File: docker/ray-ml/Dockerfile (hunk 1 above)
Comment (amogkam) [reply docker/ray-ml/Dockerfile:11]: They are already not in alphabetical order...will leave this for a future PR

Comment (amogkam) [reply docker/ray-ml/Dockerfile:11]: Actually `ml_docker` has to come at the end

Comment (amogkam) [reply docker/ray-ml/Dockerfile:11]: Ok will make the change

**Pull request merged**